lib/solidus_subscriptions/processor.rb

```diff
@@ -96,8 +96,8 @@ def new_installments(user)
           sub.advance_actionable_date
           sub.cancel! if sub.pending_cancellation?
           sub.deactivate! if sub.can_be_deactivated?
-          sub.installments.create!
-        end
+          sub.installments.create! unless sub.installments.actionable.any?
```

Comment:
I think `unless` is fine in guard clauses, but do you think `if sub.installments.actionable.empty?` is more readable? Also it might not matter in this context, but I saw this recently: https://mensfeld.pl/2015/06/empty-vs-blank-vs-any-why-you-should-not-use-any-to-test-if-theres-anything-in-the-array/